internal/service/streamconnection/data_source_stream_connections.go

```diff
@@ -42,13 +75,17 @@ func (d *streamConnectionsDS) Read(ctx context.Context, req datasource.ReadReque
 
 	connV2 := d.Client.AtlasV2
 	projectID := streamConnectionsConfig.ProjectID.ValueString()
-	instanceName := streamConnectionsConfig.InstanceName.ValueString()
+	workspaceName := getWorkspaceOrInstanceNameForDS(&streamConnectionsConfig)
```

Comment:
I'm not sure if we want it consistent.
```suggestion
	workspaceOrInstanceName := getWorkspaceOrInstanceNameForDS(&streamConnectionsConfig)
```